csi.proto

```diff
@@ -376,6 +381,10 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // This field is OPTIONAL. This allows the CO to specify the
+  // volume performance class parameters to apply. 
+  map<string, string> mutable_parameters = 8;
```

Comment:
I think this field should be marked alpha

Reply:
Marked as [(alpha_field) = true];